### PR TITLE
switch to ubuntu:bionic base image

### PR DESCRIPTION
**Pull request by chenbh**
Part of concourse/concourse#3266

Image size is now 12.6mb -> 102mb

### CODE REVIEW

**Review by cirocosta — CHANGES_REQUESTED** [review]
Hey,

I noticed that the README misses some updates regarding the fact that the Dockerfile directory can now be either `ubuntu` or `alpine`. Would you mind providing some updates to that file as well to reflect the changes?

Thanks!

File: dockerfiles/ubuntu/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  && rm -rf /var/lib/apt/lists/*
+
+ADD assets/ /opt/resource/
+RUN chmod +x /opt/resource/*
+
+FROM resource AS tests
+RUN apt-get update && apt-get install ruby-full ruby-bundler -y
+ADD . /resource
+WORKDIR /resource
+RUN ls -asl /opt/resource
```
Comment (cirocosta) [inline dockerfiles/ubuntu/Dockerfile]: Hey,

Is there a need for this `ls` here?

Thanks!

```suggestion
```

**Review by cirocosta — CHANGES_REQUESTED** [review]
Adding to the other review, this PR could:

- avoid the unnecessary listing,
- add instructions to `README` stating how one can test both Alpine and Ubuntu images

Thanks!

File: dockerfiles/ubuntu/Dockerfile (hunk 1 above)
Comment (chenbh) [reply dockerfiles/ubuntu/Dockerfile]: Yeah we also noticed that some of the other commands aren't as efficient as it could be

**Review by cirocosta — APPROVED** [review]
SGTM!

**Pull request merged**